tensorflow/examples/learn/mnist.py

```diff
@@ -62,7 +62,7 @@ def conv_model(features, labels, mode):
   # Densely connected layer with 1024 neurons.
   h_fc1 = tf.layers.dense(h_pool2_flat, 1024, activation=tf.nn.relu)
   if mode == tf.estimator.ModeKeys.TRAIN:
-    h_fc1 = tf.layers.dropout(h_fc1, rate=0.5)
+    h_fc1 = tf.layers.dropout(h_fc1, rate=0.5, training=mode == tf.estimator.ModeKeys.TRAIN)
```

Comment:
It is equivalent to `training=True`

Reply:
Not if called from evaluation.

Reply:
if called from evaluation this line won't be executed.